boards/shared/Test/Src/Test_SharedWaitSignal.cpp

```diff
@@ -0,0 +1,250 @@
+#include "Test_Shared.h"
+
+extern "C"
+{
+#include "App_SharedWaitSignal.h"
+}
+
+// TODO: Refactor fake function names after 668 is merged
```

Comment:
:eyes: :eyes: :eyes: :eyes: :eyes: :eyes:

Reply:
oh whoops forgot to delete this

Reply:
This should be deleted